src/transformState.js

```diff
@@ -6,6 +6,28 @@
  */
 function transformState(state, actions) {
   // write code here
+
+  for (const action of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```

Reply:
it was about removing blank lines and comments